README.md

```diff
@@ -326,14 +326,14 @@ https://python-statemachine.readthedocs.io.
 
 ## Contributing
 
-<a class="github-button" href="https://github.com/fgmacedo/python-statemachine" data-icon="octicon-star" aria-label="Star fgmacedo/python-statemachine on GitHub">Star this project</a>
-<a class="github-button" href="https://github.com/fgmacedo/python-statemachine/issues" data-icon="octicon-issue-opened" aria-label="Issue fgmacedo/python-statemachine on GitHub">Open an Issue</a>
-<a class="github-button" href="https://github.com/fgmacedo/python-statemachine/fork" data-icon="octicon-repo-forked" aria-label="Fork fgmacedo/python-statemachine on GitHub">Fork</a>
+* <a class="github-button" href="https://github.com/fgmacedo/python-statemachine" data-icon="octicon-star" aria-label="Star fgmacedo/python-statemachine on GitHub">Star this project</a>
+* <a class="github-button" href="https://github.com/fgmacedo/python-statemachine/issues" data-icon="octicon-issue-opened" aria-label="Issue fgmacedo/python-statemachine on GitHub">Open an Issue</a>
+* <a class="github-button" href="https://github.com/fgmacedo/python-statemachine/fork" data-icon="octicon-repo-forked" aria-label="Fork fgmacedo/python-statemachine on GitHub">Fork</a>
 
 - If you found this project helpful, please consider giving it a star on GitHub.
 
 - **Contribute code**: If you would like to contribute code to this project, please submit a pull
-  request. For more information on how to contribute, please see our [contributing.md](contributing.md) file.
+request. For more information on how to contribute, please see our [contributing.md](docs/contributing.md) file.
```

Comment:
There's an issue here: This readme.md works as the main entry point for the GitHub and the cover page on pypi, but its also included on the docs. If we fix the link for one we broke the other :(

Reply:
would it then be OK to rename [/CONTRIBUTING.md](https://raw.githubusercontent.com/fgmacedo/python-statemachine/develop/CONTRIBUTING.md) to /contributing.md?
This would work on both sites - but I don t know why the file is all upper case.

Reply:
I think that's ok. Its only for historical reasons. Accordingly to the docs, we can have it only on under the docs folder. https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/setting-guidelines-for-repository-contributors#adding-a-contributing-file